### ISSUE TITLE
Bug in RTEDIT s19 file ($0490 record) only displaying first 32 chars of long source line entered in editor

### ISSUE THREAD

**Issue by stanleyruppert** [linuxha/RT68mx#4]
Neil,
Was chasing a bug I had in RTEDIT (only displaying the first 32 characters of entered source lines having lengths between 32 and 64 characters) and found typos in the rtedit.s19 source file we typed in for $0490 record.

By sheer bad luck, the typos in the original record and the fixed record both have the same checksum!
Checksum as computed doesn't always catch multi-bit / multi-byte errors.

Here is the line for address $0490 as entered:
S11304903434 6D 0E B8 CF 1000BD0F00 4E C621374F57
and fixed 
S11304903434 BD 0E 68 CE 1000BD0F00 4F C621374F57
( I went over my original manual with a bright light and magnifier to examine each character)

Changing the above code fixed the bug and now characters of a long source line are printed (and saved).
This should resolve your disassembly issues for that code block.

Stan

![rtedit-0490-orig](https://github.com/user-attachments/assets/fb8c0530-b407-46d1-b7c9-112da409addb)

**Comment by linuxha** [comment]
Excellent, that bug was driving me nuts! I was going to recompile the redit basic code under Flex using Frederic Brown's PT68-1 (all in one SWTPC clone). I'm still going to run that through there but I think you found the bug that was causing me issues.!

Let me get this fixed up.

**Comment by linuxha** [comment]
Updated! 

A bit of a side note. I've been playing with [CC6303](https://github.com/linuxha/CC6303) and I think I have it working. I need to push my changes to my repos. I am a bit worried that I've messed it up. I ran across a lot of problems that suggest that the compiler didn't work.  I still need to create a target-s19 so the compiler can create S19 files for use in ROM monitors and RT68MX. Still need a few more libraries for the RT68MX APIs.
